Makefile

```diff
@@ -87,5 +89,8 @@ zsbl/start.o: zsbl/ux00_zsbl.dtb
 %.o: %.c $(H)
 	$(CC) $(CFLAGS) -o $@ -c $<
 
+%-board_setup.o: %.c $(H)
+	$(CC) -DBOARD_SETUP $(CFLAGS) -o $@ -c $<
+
 clean::
-	rm -f */*.o */*.dtb zsbl.bin zsbl.elf zsbl.asm fsbl.bin fsbl.elf fsbl.asm lib/version.c
+	rm -f */*.o */*.dtb {zsbl,fsbl,board_setup}.{bin,elf,asm} lib/version.c
```

Comment:
This won't work on systems where /bin/sh is dash, because brace expansion is a GNU bash feature.  I just learned that this week; we had to fix a some rules using brace expansion in riscv-gnu-toolchain.  Note that this means that it works on Ubuntu 16.04 which is on our servers, but does not work on Ubuntu 18.04 or later.